client/blocks/jetpack-benefits/index.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import {
+	isJetpackBackupSlug,
+	isJetpackScanSlug,
+	isJetpackPlanSlug,
+} from '@automattic/calypso-products';
+import getRewindState from 'calypso/state/selectors/get-rewind-state';
+import getSiteScanState from 'calypso/state/selectors/get-site-scan-state';
+import JetpackBenefitsSiteVisits from 'calypso/blocks/jetpack-benefits/site-visits';
+import JetpackBenefitsScanHistory from 'calypso/blocks/jetpack-benefits/scan-history';
+import JetpackBenefitsSiteBackups from 'calypso/blocks/jetpack-benefits/site-backups';
+import { JetpackBenefitsCard } from 'calypso/blocks/jetpack-benefits/benefit-card';
+import {
+	productHasSearch,
+	productHasBackups,
+	productHasScan,
+	productHasActivityLog,
+	productHasAntiSpam,
+} from 'calypso/blocks/jetpack-benefits/feature-checks';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	siteId: number;
+	productSlug: string;
+}
+
+// named export for cleaner testing of class methods
+const JetpackBenefits: React.FC< Props > = ( { siteId, productSlug } ) => {
+	const rewindState = useSelector( ( state ) => {
+		return getRewindState( state, siteId );
+	} );
+
+	const scanState = useSelector( ( state ) => {
+		return getSiteScanState( state, siteId );
+	} );
+	const translate = useTranslate();
+
+	const siteHasBackups = () => {
+		return 'unavailable' !== rewindState?.state;
+	};
+
+	const siteHasScan = () => {
+		return 'unavailable' !== scanState?.state;
+	};
+
+	return (
+		<React.Fragment>
+			{
+				isJetpackPlanSlug( productSlug ) && <JetpackBenefitsSiteVisits siteId={ siteId } /> // makes the most sense to show visits/ stats for plans
+			}
+			{ siteHasBackups() && productHasBackups( productSlug ) && (
+				<JetpackBenefitsSiteBackups
+					siteId={ siteId }
+					isStandalone={ isJetpackBackupSlug( productSlug ) }
+				/>
+			) }
+			{ productHasSearch( productSlug ) && (
+				<JetpackBenefitsCard
+					headline={ translate( 'Search' ) }
+					description={ translate(
+						'Jetpack Search helps your visitors instantly find the right content – right when they need it.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Incredibly powerful and customizable, Jetpack Search, helps your visitors instantly find the right content – right when they need it.' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-benefits-components)